pyproject.toml

```diff
@@ -25,7 +25,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 importlib-metadata = {version = "^4.0", python = "<3.8"}
-ansys-fluent-core = "~=0.18.dev0"
+ansys-fluent-core = ">=0.18"
```

Comment:
I think this is more appropriate now as we have now different release cadence for pyfluent and pyfluent-parametric. The pyfluent-parametric 0.9 will remain compatible with pyfluent 0.18, 0.18.1, 0.19, ... so on. After a pyfluent release in future, if pyfluent-parametric becomes incompatible we need to fix the incompatibility and make a new public release of parametric with the upgraded version of pyfluent dependency.